PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskEmcalJetSpectra8TeVTriggerQA.cxx

```diff
@@ -729,6 +748,13 @@ Bool_t AliAnalysisTaskEmcalJetSpectra8TeVTriggerQA::Run()
         }
     }
     
+    fClusterEResolution = new TF1("fClusterEResolution","sqrt([0]^2+[1]^2*x+([2]*x)^2)*0.01");
+    fClusterEResolution->SetParameters(4.35,9.07,1.63);
+    
+    //fGeom =  AliEMCALGeometry::GetInstance("EMCAL_COMPLETEV1");
+    fGeom =  AliEMCALGeometry::GetInstance("EMCAL_COMPLETE12SMV1");
```

Comment:
This should be loaded automatically based on the period. You don't need to perform it yourself (unless you are doing something special with the geometry)